src/main/java/com/devchang202/calculator/Calculator.java

```diff
@@ -0,0 +1,112 @@
+package com.devchang202.calculator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Calculator {
+    int add(int a, int b) {
+        return a + b;
+    }
+
+    int substrate (int a, int b){
+        return a - b;
+    }
+
+    int divide (int a, int b) {
+        return a / b;
+    }
+
+    int multiply(int a, int b){
+        return a * b;
+    }
+
+    String[] parseEquation(String UserInput) {
```

Comment:
```suggestion
    String[] parseEquation(String userInput) {
```

변수의 경우 카멜케이스로 작성해주는 것이 관례입니다:)